native/opencv-gpu.cc

```diff
@@ -171,63 +178,76 @@ void cv_gpu_cascade_detect(cv::Ptr<cv::cuda::CascadeClassifier>* cascade, cv::cu
     cv_to_ffi(vec_object, objects);
 }
 
-void cv_gpu_cascade_set_find_largest_object(cv::Ptr<cv::cuda::CascadeClassifier>* cascade, bool value) {
+void cv_gpu_cascade_set_find_largest_object(
+    cv::Ptr<cv::cuda::CascadeClassifier>* cascade, bool value) {
     (*cascade)->setFindLargestObject(value);
 }
 
-void cv_gpu_cascade_set_max_num_objects(cv::Ptr<cv::cuda::CascadeClassifier>* cascade, int32_t num) {
+void cv_gpu_cascade_set_max_num_objects(
+    cv::Ptr<cv::cuda::CascadeClassifier>* cascade, int32_t num) {
     (*cascade)->setMaxNumObjects(num);
 }
 
-void cv_gpu_cascade_set_min_neighbors(cv::Ptr<cv::cuda::CascadeClassifier>* cascade, int32_t min) {
+void cv_gpu_cascade_set_min_neighbors(
+    cv::Ptr<cv::cuda::CascadeClassifier>* cascade, int32_t min) {
     (*cascade)->setMinNeighbors(min);
 }
 
-void cv_gpu_cascade_set_max_object_size(cv::Ptr<cv::cuda::CascadeClassifier>* cascade, Size2i max_size) {
+void cv_gpu_cascade_set_max_object_size(
+    cv::Ptr<cv::cuda::CascadeClassifier>* cascade, Size2i max_size) {
     cv::Size cv_max_size(max_size.width, max_size.height);
     (*cascade)->setMaxObjectSize(cv_max_size);
 }
 
-void cv_gpu_cascade_set_min_object_size(cv::Ptr<cv::cuda::CascadeClassifier>* cascade, Size2i min_size) {
+void cv_gpu_cascade_set_min_object_size(
+    cv::Ptr<cv::cuda::CascadeClassifier>* cascade, Size2i min_size) {
     cv::Size cv_min_size(min_size.width, min_size.height);
     (*cascade)->setMinObjectSize(cv_min_size);
 }
 
-void cv_gpu_cascade_set_scale_factor(cv::Ptr<cv::cuda::CascadeClassifier>* cascade, double factor) {
+void cv_gpu_cascade_set_scale_factor(
+    cv::Ptr<cv::cuda::CascadeClassifier>* cascade, double factor) {
     (*cascade)->setScaleFactor(factor);
 }
 
-Size2i cv_gpu_cascade_get_classifier_size(cv::Ptr<cv::cuda::CascadeClassifier>* cascade) {
+Size2i cv_gpu_cascade_get_classifier_size(
+    cv::Ptr<cv::cuda::CascadeClassifier>* cascade) {
     cv::Size2i size = (*cascade)->getClassifierSize();
-    Size2i c_size = {.width = size.width, .height = size.height };
+    Size2i c_size = {.width = size.width, .height = size.height};
```

Comment:
This syntax is GCC-only, it won't work on MSVC. We don't compile GPU modules on AppVeyor, this is why this thing happen. I think it should be fixed. Correct syntax for both compilers is `Size2i c_size = {size.width, size.height};`. I had the same issue with `CVec`.

Reply:
Thanks for catching this. Fixed.

Reply:
@nebgnahz No problem. I think there are some other places too. Lines 202, 220, 226.